podo/PodoCount_code/podocount_main_serv.py

```diff
@@ -144,18 +152,25 @@
         x_start,y_start,x_stop,y_stop = int(df2*bb[0]),int(df2*bb[1]),int(df2*bb[2]),int(df2*bb[3])
         x_length = x_stop-x_start
         y_length = y_stop-y_start
-
         roi = np.array(WSI_file.read_region((y_start,x_start),0,(y_length,x_length)),dtype = 'uint8')
         roi = roi[:,:,0:3]
         rows,cols,dims = roi.shape
         roi_name = '/roi_' + str(bb_iter+1) + '.png'
         roi_path = wsi_roi_dir + roi_name
-        plt.imsave(roi_path,roi)
-
         glom_mask = WSI_glom_mask[bb[0]:bb[2],bb[1]:bb[3]]
-        glom_mask = (resize(glom_mask,[rows,cols],anti_aliasing=True)>0.01)*1
-
-        xml_counter, xml_contour, gcount, pcount, glom_pod_feat_vector, indv_pod_feats = get_pod_props(roi,glom_mask,slider,x_start,y_start,xml_counter,xml_contour, gcount, pcount,dist_mpp,area_mpp2, section_thickness, wsi_pod_seg_dir)
+        glom_mask_uint8 = (glom_mask * 255).astype(np.uint8)
+        pil_image_glom = Image.fromarray(glom_mask_uint8)
+        print(pil_image_glom.size, "before pil image")
+        width, height = glom_mask.shape
+        glom_mask = pil_image_glom.resize((height, width), Image.Resampling.LANCZOS)
+        glom_mask = np.array(glom_mask)
+        glom_mask = glom_mask // 255
+
+        if args.type == 'Human_Analysis':
+            xml_counter, xml_contour, gcount, pcount, glom_pod_feat_vector, indv_pod_feats = get_pod_props(roi,glom_mask,slider,x_start,y_start,xml_counter,xml_contour, gcount, pcount,dist_mpp,area_mpp2, section_thickness, wsi_pod_seg_dir)
+        elif args.type == 'Mouse_Analysis':

```

Comment:
I would also recommend to use a `enum` class here. See [this](https://docs.python.org/3/library/enum.html) documentation. You can define the class on top of this file as (feel free to change the names):
```
from enum import Enum
class InputType(Enum):
    Mouse_Analysis = 'Mouse'
    Human_Analysis = 'Human'
```

And then you can check each type as:
`args.type == InputType.Mouse_Analysis.value` and `args.type == InputType.Human_Analysis.value`